src/ClimaAtmos/Dycore/src/VanillaAtmosDiscretizations.jl

```diff
@@ -186,12 +188,23 @@ function estimatedt(::Val{dim}, ::Val{N}, G, gravity, Q, vgeo,
 
   dt = [floatmax(DFloat)]
 
+  #Allocate at least three spaces for qm, with a zero default value
+  #q_m = zeros(DFloat, 3)
+    
   if dim == 2
     @inbounds for e = 1:nelem, n = 1:Np
       ρ, U, V = Q[n, _ρ, e], Q[n, _U, e], Q[n, _V, e]
       E = Q[n, _E, e]
       y = vgeo[n, G.yid, e]
-      P = gdm1*(E - (U^2 + V^2)/(2*ρ) - ρ*gravity*y)
+        
+      #Moist tracers
+      #for m = 1:nmoist
+      #    s = _nstate + m
+      #    q_m[m] = Q[n, s, e]
+      #end
+      #(R_m, cp_m, cv_m, gamma_m) = MoistThermodynamics.moist_gas_constants(q_m[1], q_m[2], q_m[3])        
+      #gdm1 = R_m/cv_m
+      P    = gdm1*(E - (U^2 + V^2)/(2*ρ) - ρ*gravity*y)
```

Comment:
Use moist thermodynamics functions for this. This is air pressure for `T=T_0`--pretty strange anyway. Maybe add a function to `MoistThermodynamics' to extract internal energy from total, if we need this.